src/styles/blocks/card.css

```diff
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__title {
+  color: var(--main-color);
+  line-height: 18px;
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.card__code {
+  color: var(--text-color);
+  margin-bottom: 16px;
+}
+
+.card__block {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__review {
+  color: var(--main-color);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  color: var(--text-color);
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__sum {
+  color: var(--main-color);
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
```

Comment:
You may not use width here, should work without it

Reply:
done